view/frontend/templates/page/analytics.phtml

```diff
@@ -8,7 +8,21 @@ $events = $block->getEvents();
 ?>
 <script type="text/javascript">
     require(['<?php echo $url; ?>'], function(){
```

Comment:
If magento2 has `requireJS` and we use it for async script loading, it seems that the "events code" will be executed after our library is loaded. If this is the case, we don't need the "metrilo queue" code in here.

Reply:
Reverting it.